include/onnxruntime/core/graph/graph.h

```diff
@@ -394,6 +398,9 @@ class Node {
   /** Gets the Node's attributes. */
   const NodeAttributes& GetAttributes() const noexcept { return attributes_; }
 
+  /** @returns true if the Node is a forward node, false otherwise. **/
+  bool isForwardNode() const noexcept { return isForwardNode_; }
```

Comment:
Can the default compiled binaries even train a model or is a specialized compilation needed ?